### PR TITLE
incorporated ability to view items

### PR DESCRIPTION
**Pull request by Ruizhi2001**
Now we can view items. Any other command will bring you back to home

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -197,14 +220,14 @@ private CommandResult executeCommand(String commandText) throws CommandException
...
                 handleExit();
-            }
-
-            if (commandResult.isStallDetail()) {
+            } else if (commandResult.isStallDetail()) {
                 handleIsStallDetail();
+            } else if (commandResult.isViewItem()) {
+                handleItemSelectionChanged(logic.getFilteredItem());
+            } else {
+                this.backToHomePage();
```
Comment (teoks0199) [inline src/main/java/seedu/address/ui/MainWindow.java:230]: 👍

**Review by teoks0199 — APPROVED** [review]
lgtm but can add test classes for view-item next time

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (jamesebond) [reply src/main/java/seedu/address/ui/MainWindow.java:230]: VELI VELI LGTM WOW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java
Comment (jamesebond) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:1]: LGTMMMMMMM LEEEE

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -49,21 +49,27 @@
...
-            <padding>
-              <Insets top="10" right="10" bottom="10" left="10" />
-            </padding>
-            <StackPane fx:id="stallDetailsPanelPlaceholder" VBox.vgrow="ALWAYS"/>
-          </VBox>
-        </items>
+
+          <items>
+
+            <VBox fx:id="leftMainPanel" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="NEVER">
```
Comment (beatricecst) [inline src/main/resources/view/MainWindow.fxml:56]: LGTM

**Review by beatricecst — APPROVED** [review]

**Review by jamesebond — APPROVED** [review]

**Pull request merged**